fuzzers/coverage/patch.diff

```diff
@@ -0,0 +1,55 @@
+diff --git a/compiler-rt/lib/fuzzer/FuzzerDriver.cpp b/compiler-rt/lib/fuzzer/FuzzerDriver.cpp
+index 306e644277c..26db440ecd7 100644
+--- a/compiler-rt/lib/fuzzer/FuzzerDriver.cpp
++++ b/compiler-rt/lib/fuzzer/FuzzerDriver.cpp
+@@ -737,6 +737,8 @@ int FuzzerDriver(int *argc, char ***argv, UserCallback Callback) {
+       F->SetMaxInputLen(kDefaultMaxMergeLen);
+     assert(Flags.merge_control_file);
+     F->CrashResistantMergeInternalStep(Flags.merge_control_file);
++    if (Options.DumpCoverage)
```

Comment:
I'm not sure I understand this. If timeout happens inside `CrashResistantMergeInternalStep` and it crashes, coverage data inside if will be lost. Did you try to convert `.profraw` that you're getting locally into `.profdata`? Do they have non-zero counters? The best way to verify that is to check the number of execution on `LLVMFuzzerTestOneInput`

Reply:
Sorry, I'm not very familiar with the llvm-project structure. I will dig deeper to see what will happen if it times out inside CrashResistantMergeInternalStep. I did the DumpCoverage here because I saw @jonathanmetzman did the same for sanitizer coverage. And the .profdata works fine. It has non-zero counters and execution number on LLVMFuzzerTestOneInput is correct.